backend/src/main/resources/templates/Rhosak/actionRequiredBody.html

```diff
@@ -0,0 +1,2 @@
+{#include Rhosak/emailBodyHeader}
```

Comment:
CI is complaining because all `{#include ...}` tags must be closed with `{/include}`.

Reply:
I am looking at this now. I want to put some skeleton in the initial templates. 

My question was on the following exception
```
Couldn't persist aggregation!: javax.validation.ConstraintViolationException: Validation failed for classes [com.redhat.cloud.notifications.models.EmailAggregation] during persist time for groups [javax.validation.groups.Default, ]
List of constraint violations:[
	ConstraintViolationImpl{interpolatedMessage='must not be null', propertyPath=accountId, rootBeanClass=class com.redhat.cloud.notifications.models.EmailAggregation, messageTemplate='{javax.validation.constraints.NotNull.message}'}
]
```

or it is not something I've to worry about?

Reply:
Don't worry about this warn, it's not related to your PR. It comes from here:
https://github.com/RedHatInsights/notifications-backend/blob/2ed05771a0a37a74518c1896c5e7c0cfd9f5af2f/backend/src/test/java/com/redhat/cloud/notifications/db/EmailAggregationResourcesTest.java#L81-L92

Reply:
Perfect, thanks. I guess we are ready to promote the PR for full review. Thank you again for the initial review @gwenneg